API & Team management updates (#48)
quantcdn/docs: docs/dashboard/team.md

Only organization owners or administators have permission to manage team members.

### Inviting new members

Click on the "Invite" button and provide name, email and role to add users to the organization.

You may revoke the invitation before they accept.

### Accepting or declining an invitation

Users will receive an email prompting them to join. If they already have an account, they can click the link and login to accept; otherwise they must first register.

Inivitations and existing memberships are also listed from the profile page. They can be accepted or declined from this screen.

## Change active organizations

Click on your profile icon (upper right) to switch between organizations and your own private account.

Projects are scoped to organizations. Once you have activated an organization, the projects list (upper left) can be used to toggle the active project.

### Enforcing two-factor authentication

The organization owner can enforce two-factor authentication by checking the box at the top of the page.

Users will be prompted to complete the two-factor authentation set up before they can undertake any further action within the dashboard.

### Changing roles

> To change an existing user role you must have an organization owner or administrator remove and re-add you with the appropriate role.

Author: ```suggestion
To change an existing user role an organization owner or administrator must remove and re-add you with the appropriate role.
```

---

> id: team

Reviewer: Double check the use of `##` vs `###` in this doc as it seems like `###` is used when it should be `##`, i.e. all should probably be at the same level... otherwise the nested seems odd.

Author: Good call, all updated to `##`

title: Team management
sidebar_label: Team management
---

## Overview

Organization subscriptions allow multiple team members to belong to a single organization. Roles can be assigned to control access.

Navigate to the "Team" section of the dashboard to see all current members and their roles.

Only organization owners or administators have permission to manage team members.

### Inviting new members

Click on the "Invite" button and provide name, email and role to add users to the organization.

You may revoke the invitation before they accept.

### Accepting or declining an invitation

> Users will receive an email prompting them to join. If they already have an account they can click the link and login to accept, otherwise they must first register.

Reviewer: Nitpick: Add comma after "have an account"

"accept, otherwise" => "accept; otherwise,"

> Users will receive an email prompting them to join. If they already have an account they can click the link and login to accept, otherwise they must first register.

Author: ```suggestion
Users will receive an email prompting them to join. If they already have an account, they can click the link and login to accept; otherwise they must first register.
```

> Inivitations and existing memberships are also listed from the profile page - they can be accepted or declined from this screen.

Reviewer: "page - they" => "page. They"

> Inivitations and existing memberships are also listed from the profile page - they can be accepted or declined from this screen.

Author: ```suggestion
Inivitations and existing memberships are also listed from the profile page. They can be accepted or declined from this screen.
```

## Change active organizations

> Click on your profile icon (upper right) to switch between organizations or your own private account.

Reviewer: Nitpick: "or your own" => "and your own"

> Click on your profile icon (upper right) to switch between organizations or your own private account.

Author: ```suggestion
Click on your profile icon (upper right) to switch between organizations and your own private account.
```

> Projects are scoped to organizations. Once you have activated an organization the projects list (upper left) can be used to toggle the active project.

Reviewer: Nitpick: "organization the projects" => "organization, the projects"

> Projects are scoped to organizations. Once you have activated an organization the projects list (upper left) can be used to toggle the active project.

Author: ```suggestion
Projects are scoped to organizations. Once you have activated an organization, the projects list (upper left) can be used to toggle the active project.
```

### Enforcing two-factor authentication

The organization owner can enforce two-factor authentication by checking the box at the top of the page.

> Users will be prompted to complete the two-factor authentation enrolment before they can undertake any further action within the dashboard.

Reviewer: enrolment => set up

> Users will be prompted to complete the two-factor authentation enrolment before they can undertake any further action within the dashboard.

Author: ```suggestion
Users will be prompted to complete the two-factor authentation set up before they can undertake any further action within the dashboard.
```

### Changing roles

To change an existing user role you must have an organization owner or administrator remove and re-add you with the appropriate role.

> 

Reviewer: Nitpicks:

"To change an existing user role you must have an organization => "To change an existing user role, an organization  owner or administrator must remove"

Extra blank line

> 

Author: ```suggestion
```

| View crawler config        | ✅          |                  | ✅        | ✅            | ✅                  |
| Create crawler             |             |                  |           | ✅            | ✅                  |
| Edit crawler config        | ✅          |                  |           | ✅            | ✅                  |
| Trigger crawls             | ✅          |                  |           | ✅            | ✅                  |
| View crawl history         | ✅          |                  | ✅        | ✅            | ✅                  |
| View team members          | ✅          | ✅               | ✅        | ✅            | ✅                  |
| Manage team members        |             |                  |           | ✅            | ✅                  |
| Manage subscription        |             |                  |           | ✅            | ✅                  |
| Enforce TFA                |             |                  |           |               | ✅                  |
| Reset team member TFA¹     |             |                  |           |               | ✅                  |

> ¹ via support desk. Additional verification processes apply.

Reviewer: Nitpick: "via support desk" => "Requested via the support desk"

> ¹ via support desk. Additional verification processes apply.

Author: ```suggestion
¹ via the support desk. Additional verification processes apply.
```
